todo/src/App.scss

```diff
@@ -0,0 +1,106 @@
+$redux-purple : #764ABC;
+$react-blue: #282c34;
+
+.App {
+  text-align: center;
+}
+
+figure {
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+
+  div {
+    display: flex;
+    flex-direction: column;
+    height: 100px;
+    padding: 10px;
+  }
+  img {
+    width: 100px;
+    align-self: center;
+    margin: 10px auto;
+  }
+
+  a {
+    margin-top: 20px;
+  }
+}
+
+img {
+  width: 100px;
+  display: inline-block;
+  margin: 10px auto;
+
+}
+
+ul {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  border: 5px solid $redux-purple;
+  padding-left: 0px;
+}
+
+.todo {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px;
+  border: 5px solid white;
+  font-size: 25px;
+  width: 30%;
+  background-color: $redux-purple;
+  border-radius: 10px;
+  color: white;
+  // border: 3px solid black;
+  // margin: 5px;
+
+    
+    li {
+    display: inline-block;
+    }
+
+}
+
+form {
+  padding: 10px;
+  margin-top: 20px;
+  // border: 1px solid red;
+
+
+  input {
+    font-size: 20px;
+    padding: 10px;
+  }
+
+}
+
+button {
+  margin-left: 10px;
+  padding: 10px;
+}
+
+.App-header {
+  background-color: $redux-purple;
+  // min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+
+.App-link {
+  color: #61dafb;
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
```

Comment:
Glad to see you using scss, great job on the styling. The hosted app looks really good